packages/elements-dev-portal/.storybook/preview.jsx

```diff
@@ -44,7 +46,11 @@ const MosaicProviderDecorator = (Story) => (
   <MosaicProvider><Story/></MosaicProvider>
 );
 
-export const decorators = [ThemeProvider, ProviderDecorator, MosaicProviderDecorator, PersistenceBoundaryDecorator];
+const DevPortalProviderDecorator = (Story, context) => {
+  return (<Provider platformUrl={context.args.platformUrl}><Story/></Provider>);
```

Comment:
Dev-portal storybook runs better with that change :wink:
```suggestion
  return (<DevPortalProvider platformUrl={context.args.platformUrl}><Story/></DevPortalProvider>);
```

It might also require adding `QueryClientProvider` here in storybook

Reply:
Fixed with https://github.com/stoplightio/elements/pull/1193/commits/6426e2ad7cb9c8c046dcdd86784ea4e14374c5f9

Reply:
Does it work for you without `QueryClientProvider`? I had to add it to make it not throw
```ts
const QueryClientProviderDecorator = (Story) => {
  return (<QueryClientProvider client={queryClient}><Story/></QueryClientProvider>);
}
```

Reply:
Oh good catch, forgot I refactored that out of the DevPortalProvider. I'll add that decorator, thanks!